src/main/java/uk/gov/justice/probation/courtcasematcher/service/CourtCaseService.java

```diff
@@ -62,6 +62,12 @@ public void saveCourtCase(CourtCase courtCase) {
         restClient.putCourtCase(courtCase.getCourtCode(), courtCase.getCaseNo(), courtCase);
     }
 
+    public Mono<CourtCase> updateProbationStatusDetail(CourtCase courtCase) {
```

Comment:
This is the bit that will get and merge the probation status detail fields on cases that we are updating